Doxyfile

```diff
@@ -14,3 +17,18 @@ GENERATE_LATEX         = NO
 GENERATE_HTML          = YES
 
 SORT_MEMBERS_CTORS_1ST = YES
+
+GENERATE_TREEVIEW      = YES
+DISABLE_INDEX          = NO
+FULL_SIDEBAR           = NO
+HTML_HEADER            = ../docs/doxygen-awesome-theme/header.html
+HTML_EXTRA_STYLESHEET  = ../docs/doxygen-awesome-theme/doxygen-awesome.css \
+                         ../docs/doxygen-awesome-theme/doxygen-awesome-sidebar-only.css \
+                         ../docs/doxygen-awesome-theme/doxygen-awesome-sidebar-only-darkmode-toggle.css 
+HTML_EXTRA_FILES       = ../docs/doxygen-awesome-theme/doxygen-awesome-darkmode-toggle.js \
+                         ../docs/doxygen-awesome-theme/doxygen-awesome-interactive-toc.js
+
+HTML_COLORSTYLE        = LIGHT
```

Comment:
Is it light theme by default?

Reply:
In the theme it actually says that this is required to be set to LIGHT :rage4: 
For me it shows dark theme by default. I think it takes your system settings tbh.